Zendro_requests_with_python.md

```diff
@@ -0,0 +1,246 @@
+# How to use the Zendro API with python to make changes to the data
+
+If you are a user with CRUD credentials you can use Zendro GraphQL API to directly make changes to the data. 
+
+Using the Zendro API its mostly straightforward and similar to how we write queries in [GraphiQL](https://zendro.conabio.gob.mx/graphiql) (check the [GraphQL_intro.md](GraphQL_intro.md)), but with some extra steps.
+
+The first thing we need to do is to get a session token so we can authenticate our queries. To achieve this, we use the python library `requests`. The steps to get a session token and use it with our requests are the following:
```

Comment:
Maybe a short word of advise, that this token will expire after 30 minutes.

Reply:
Good point. Done. 7631755